src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_ONE_DAY = 40;
+  const LONG_TERM = 7;
```

Comment:
Just in lowerCase or camelCase
Do it for constant wich you reuse in a lot of files
```suggestion
  const priceOneDay = 40;
  const long_term = 7;
```